src/js/utils/events.js

```diff
@@ -351,8 +351,12 @@ export function off(elem, type, fn) {
 
   // Are we removing all bound events?
   if (!type) {
-    for (const t in data.handlers) {
-      removeType(t);
+    if (!!data && !!data.handlers) {
```

Comment:
Why do this check here? If `data.handlers` doesn't exist, we should already have exited on line 338.

Reply:
You are right. It appears as if I was overly cautious when looking for the reason of ```data.handlers``` being ```undefined```.